### DISCUSSION TITLE
Request for Community Standards section to be filled out

### DISCUSSION THREAD

**Discussion by eglavin** [elrumordelaluz/reactour#449] [General]
Hi there I'm just looking to see if you can add a contributing and a code of conduct to your communities standards. These are the types of things that we need to state when reviewing external libraries to be able to use.
Communities Standards:
https://github.com/elrumordelaluz/reactour/community
Also a security policy would be a good idea as well
https://github.com/elrumordelaluz/reactour/security/policy

For example:
https://github.com/gilbarbara/react-joyride/blob/main/CONTRIBUTING.md

**Comment by elrumordelaluz** [comment]
Hi @eglavin, thank you for the input! Should be available now most of the _community_ files.
Thanks again,

**Reply by eglavin** [reply]
Appreciate it, thanks for your time!
